client/dashboard/plugins/plugin/components/action-render-modal-wrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { Modal } from '@wordpress/components';
+import { ComponentProps } from 'react';
+import ActionRenderModal, {
+	ActionRenderModalProps,
+} from '../../manage/components/action-render-modal';
+
+type ActionRenderModalWrapperProps = ActionRenderModalProps &
+	Pick< ComponentProps< typeof Modal >, 'onRequestClose' | 'title' > & { isOpen: boolean };
+
+export const ActionRenderModalWrapper = ( {
+	actionId,
+	closeModal,
+	isOpen = false,
+	items,
+	onActionPerformed,
+	onExecute,
+	onRequestClose,
+	title,
+}: ActionRenderModalWrapperProps ) => {
+	return (
+		isOpen && (
```

Comment:
I think this should be placed outside, similar to how we handle normal modals, and used like:

```js
{ isOpen && <ActionRenderModalWrapper /> }
```